dotcom-rendering/src/components/YoutubeAtom/YoutubeAtom.stories.tsx

```diff
@@ -336,99 +364,130 @@ export const PausesOffscreen = {
 	},
 } satisfies Story;
 
-export const NoConsentWithIma = {
-	...NoConsent,
-	args: {
-		...NoConsent.args,
-		enableIma: true,
-	},
-} satisfies Story;
+/**
+ * Duplicate YoutubeAtom tests but with ads enabled.
+ *
+ * The YouTube ads integration (IMA) only works on allow listed https domains.
+ * The tests require running:
+ * - DCR as normal on port 3030
+ * - scripts/nginx/setup.sh to setup nginx on https://r.thegulocal.com
+ *
+ * The ad enabled tests are a convenience for manual testing.
+ *
+ */
```

Comment:
Disabled snapshots for the ads enabled tests as they require an allow listed https domain